lib/llm/message.rb

```diff
@@ -4,9 +4,18 @@ module LLM
   class Message
     attr_accessor :role, :content
 
-    def initialize(role, content)
+    def initialize(role, content, extra = {})
       @role = role
       @content = content
+      @extra = extra
+    end
+
+    ##
+    # @see https://platform.openai.com/docs/api-reference/chat/object#chat/object-choices logprobs docs
+    # Returns log probabilities
+    # @return [Hash]
+    def logprobs
```

Comment:
@antaz Can we add a `@note` ? Can we also link to the cookbook: https://cookbook.openai.com/examples/using_logprobs ?
```ruby
# @note logprobs are specific to OpenAI 
```

Reply:
Yup